examples/fundamentals/data_container.myst.md

```diff
@@ -99,20 +97,20 @@ Furthermore, inside `idata`, PyMC has automatically saved the observed (endogeno
 idata.observed_data
 ```
 
-In this next model, we create a `pm.ConstantData` containers to hold the observations, and pass this container to the `observed`. We also make a `pm.ConstantData` container to hold the `x` data:
+In this next model, we create a `pm.Data` container to hold the observations, and pass this container to the `observed`. We also make a `pm.Data` container to hold the `x` data:
```

Comment:
```suggestion
In this next model, we create a {class}`pm.Data` container to hold the observations, and pass this container to the `observed`. We also make a {class}`pm.Data` container to hold the `x` data:
```

Reply:
Change everywhere we use `pm.Data`.

Reply:
Thank you for clarifying that for me. I was not sure if the `{class}` was only supposed to appear the first time we introduce the class. I will make the changes to fix it.

Reply:
It should be one of ``{class}`pymc.Data` `` (will render pymc.Data as link) or ``{class}`~pymc.Data` `` (will render only Data as link). Sphinx doesn't know about `pm`. I prefer the 1st one.

Reply:
@twiecki @OriolAbril Would it be preferable to change all `pm.Data` to `{class}pymc.Data` or is it preferred that I go back to keeping it as `pm.Data` without `{class}`? At the start of the notebook, there is a reference to `{class}pymc.Data`.

Reply:
I personally prefer `pymc.Data` as a link, but I don't really care much either way

Reply:
Okay, I went ahead and changed it so that it is {class} `pymc.Data` everywhere in the markdown text.